build/yamls/base/crds.yml

```diff
@@ -187,6 +187,86 @@ spec:
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
+metadata:
+  name: ippools.crd.antrea.io
+spec:
+  group: crd.antrea.io
+  versions:
+    - name: v1alpha2
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          required:
+            - spec
+          properties:
+            spec:
+              type: object
+              required:
+                - ipVersion
+                - ipRanges
+              properties:
+                ipVersion:
+                  type: integer
+                ipRanges:
+                  type: array
+                  items:
+                    type: object
+                    required:
+                      - gateway
+                      - prefixLen
+                    properties:
+                      gateway:
+                        type: string
+                        oneOf:
+                          - format: ipv4
+                          - format: ipv6
+                      prefixLen:
+                        type: integer
+                      start:
+                        type: string
+                        oneOf:
+                          - format: ipv4
+                          - format: ipv6
+                      end:
+                        type: string
+                        oneOf:
+                          - format: ipv4
+                          - format: ipv6
+                      vlan:
+                        type: string
+            status:
```

Comment:
It seems this can't work. As the IPPoolStatus types defined in pkg/apis/crd/v1alpha2/types.go.
Suggested change:
```
            status:
              type: object
              properties:
                usage:
                  items:
                    type: object
                    required:
                      - ipAddress
                      - state
                      - resource
                    properties:
                      ipAddress:
                        type: string
                        oneOf:
                          - format: ipv4
                          - format: ipv6
                      state:
                        enum:
                          - Allocated
                          - Preallocated
                        type: string
                      resource:
                        type: string
                  type: array
```

Reply:
Thank you for noticing this! Fixing